tests/integration/test_pytket_circuits.py

```diff
@@ -289,20 +290,45 @@ def foo(reg: array[qubit, 2]) -> array[bool, 2]:
 
 
 @pytest.mark.skipif(not tket2_installed, reason="Tket2 is not installed")
-def test_measure_afterwards(validate, run_int_fn):
+def test_symbolic_exec(validate, run_int_fn):
+    from pytket import Circuit, OpType
+    from pytket.passes import AutoRebase
+    from sympy import Symbol
+
+    a = Symbol("alpha")
+
+    circ = Circuit(1)
+    circ.Rx(a, 0)
+    circ.measure_all()
+
+    AutoRebase({OpType.CX, OpType.Rz, OpType.H}).apply(circ)
+
+    flip = guppy.load_pytket("flip", circ, use_arrays=False)
+
+    @guppy
+    def main() -> int:
+        q = qubit()
+        res = int(flip(q, 0))
```

Comment:
So this test fails if you set `alpha = pi`?

Reply:
Ahh I guess it's because we're using the coin-flip simulator!

Reply:
Hm no I am using `Quest` if qubits are involved - I was using `comptime(math.pi)` (because it's currently floats and not angles), does that make a difference?